composer.json

```diff
@@ -16,12 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
-        "symfony/console": "^5.0"
+        "php": "^8.0",
+        "symfony/console": "^6.0"
```

Comment:
This makes it incompatible with any other version, and there is apparently no reason to lock it. The same for the PHP version. It would be better to require the following versions :

```
"require" : {
    "php": "^7.0 || ^8.0",
    "symfony/console": "^3.0 || 4.0 || 5.0 || 6.0"
}
```

It should be obviously tested against any breaking change between versions.

Reply:
I partially agree about that from my point of view.

- About the outdated version, you're completely right about migrating. It is really bad to have outdated Symfony or PHP running...

- ...but it shouldn't be the maintainer responsibility to force people update their library. If a specific feature is required by a more recent PHP version or symfony console, I agree that the maintener shouldn't bother maintaining for older versions. But if it works, there is no need to lock it. I (very) quickly went through the code, and I don't see anything that justify locking the version.

- About the PHP-CS-Fixer, it is used only for development purpose. I don't agree that it should lock the version in production. The only reason it can block it would be for the CI, that is why PHP CS Fixer in its Readme advice to install it in the `tools/php-cs-fixer` folder to avoid conflicting with the environment.

- But everything is fine if old versions are still accessible. We can consider that outdated installation won't evolve and therefore, will not need any future fix provided by future updates. 

Thanks for the conversation though, I'm always glad to talk about that :)

Reply:
Yeah you are right with the PHP-CS-Fixer. This should only run in the CI or during development.
I also agree with the locking of the versions. Just to say that the current version is already only support Symfony 5 so it might be possible to at least support that version.

In the end the decision lies with @BrainMaestro so he can decide what he wants to support.

Reply:
I believe it's fine to update the versions. Users on EOL'd versions of PHP or Symfony Console should use older versions of the package